opentelemetry-sdk/src/opentelemetry/sdk/environment_variables.py

```diff
@@ -406,3 +406,18 @@
 provide the entry point for loading the log emitter provider. If not specified, SDK
 LogEmitterProvider is used.
 """
+
+_OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE = (
```

Comment:
should this not be public as it's part of the spec?

Reply:
Not yet, since metrics has not been released. Just like the log environment variable is private still, since logs has not yet been released too.